src/commands/cmd_bit.cc

```diff
@@ -113,6 +118,15 @@ class CommandBitCount : public Commander {
       stop_ = *parse_stop;
     }
 
+    if (args.size() == 5) {
+      if (util::EqualICase(args[4], "BYTE")) {
+      } else if (util::EqualICase(args[4], "BIT")) {
```

Comment:
BIT modifier is supported in Redis 7.0.

But we assume counting in bytes everywhere so it's non-trivial to implement. It can be a follow-up issue.

Reply:
https://redis.io/commands/bitcount/